runtime/src/identity.rs

```diff
@@ -568,6 +533,40 @@ decl_module! {
                 Err("No did linked to the user")
             }
         }
+
+
+        // Manage Authorizations to join to an Identity
+        // ================================================
+
+        pub fn authorize_join_to_identity(origin, id: IdentityId) -> Result {
```

Comment:
What does this function do when called if multiple DIDs previously pre-added this account to their DIDs?

Reply:
This function keeps the 1-to-1 constraint in relation between keys and identities.
In that case, it will rise up an error with the following message: `Key is already linked to an identity`.

Reply:
What I meant is:
1- DID 1 adds key XYZ to their DID 1 (adds to pre auth list)
2- DID 2 adds key XYZ to their DID 2 (adds to pre auth list)

Assuming both operations happen one after the other before XYZ has done anything.
What happens when XYZ calls `authorize_join_to_identity()`? Which DID will they join?

Reply:
@pabloruiz55 The second parameter `id` of `authorize_join_to_identity` is the identity that `origin` (XYZ) is going to accept. 
It is confused, let me add a comment and a better parameter name.